### ISSUE TITLE
MobileMiniBrowser doesn't work correctly on iOS 12

### ISSUE THREAD

**Issue by burd1** [UInt2048/CyberKit#15]
Mobileminibrowser fails to display any page on my iPhone X with iOS 12.4 despite having all the needful configurations.

https://github.com/UInt2048/CyberKit/assets/5015897/dce4a68e-c3e0-4ff0-977b-3f9433ed8509

**Comment by UInt2048** [comment]
Can you please provide your jetsam configuration plist?

**Comment by burd1** [comment]
> Can you please provide your jetsam configuration plist?

[com.apple.jetsamproperties.D221edit.zip](https://github.com/UInt2048/CyberKit/files/12111621/com.apple.jetsamproperties.D221edit.zip)

**Comment by shenhuanchao** [comment]
Same issue here, iPhone X on ios 12.1.2 and nothing is loading.

**Comment by UInt2048** [comment]
@shenhuanchao Wait, ref609 has this problem?!

**Comment by burd1** [comment]
@UInt2048 literally none of them work my man

**Comment by UInt2048** [comment]
@burd1 Even ref607 does this?

**Comment by burd1** [comment]
> @burd1 Even ref607 does this?

Yes, i thought i had already told this to you? maybe not

**Comment by shenhuanchao** [comment]
Yeah neither 609 nor 607 loads websites. A weird URL shows up for a split second when I open the app and when I spam the “go” button, other than that it’s all blank no matter what address is inputted.

**Comment by UInt2048** [comment]
@burd1 @shenhuanchao

By any chance, are you using Chimera?

**Comment by burd1** [comment]
> @burd1 @shenhuanchao
> 
> By any chance, are you using Chimera?

Nope, using Unc0ver.

**Comment by shenhuanchao** [comment]
Also Unc0ver here, 4.3.1 to be exact.

**Comment by Tweaker177** [comment]
I have the same issue with iOS 14.2 and the latest nightly browser install 0.0.8 nightly 08.  I didn’t do any additional config after install, though. Is there something else that needs done besides installing the deb? (First time trying the tool. Sorry if i missed a ReadMe or something simple.)

**Comment by UInt2048** [comment]
> the latest nightly browser

@Tweaker177, you are affected by issue #21.

**Comment by burd1** [comment]
@UInt2048 any news on new ios 12 test builds to make it work?

**Comment by UInt2048** [comment]
@burd1 The next build for iOS 12 will likely be v0.0.9 alpha, since the work on that release might affect this issue.

**Comment by theogbob** [comment]
I still have isses, performed all the steps like jetsam and tried 0.0.8 all ref versions and 0.0.9 with the same issue as OP
